### DISCUSSION TITLE
A way to zoom and drag-select on a touch screen, without a mouse

### DISCUSSION THREAD

**Discussion by lucklobster** [Comfy-Org/ComfyUI#1139] [Ideas]
It would be helpful if there was a way to zoom the workflow diagram if you don't have a mouse or a mouse wheel, such as on a tablet touch screen.
An available zoom menu item or an on screen slider maybe?
Even having a keyboard shortcut would be good, such as being able to use the up/down arrow keys.
It would also be nice to have a way to drag select on a tablet touch screen when you don't have a mouse.

**Comment by medbukadnezoor** [comment]
I second this please... I just tried it today and my vertical mouse went haywire, can't get it to zoom in or out slowly

**Comment by FlashlightET** [comment]
yeah my scroll wheel suddenly broke (middle button still works) and now i can't zoom in

**Comment by jamsdrak** [comment]
Any Update?? i myself also have always a broken wheel , it frustrating to try comfyUI just because of that. You could implement a button layout or zoom (+, -, fitall), or implement the normal controls of other app like Maya, blender, Photoshop, Krita, etc. That implemenmt the use of Drag Zoom, using a specific key then drag mouse to change Zoom. 
Thx, in advance, great work soo far with the App. 
Still missing some basic functions like navigation and zoom controls D'oh. 

Anyway to implement at least this for now https://codepen.io/burnaDLX/pen/QWMzwGj  for the zoom  ?¿?

edit: i just check the code in litegraph.core.js aroudn 5000 line i see some function for DragAndScale, checking how it could be implemented or write something to it
..

**Reply by nntb** [reply]
I'm curious and well. I use comfy ui on my Galaxy fold 4. The lack of a pinch too zoom makes navigating so very terrible.
